server/data/manageUsersApiClient.test.ts

```diff
@@ -0,0 +1,37 @@
+import nock from 'nock'
```

Comment:
When considering swapping HMPPS Auth to the Manage Users API endpoint, I noticed that there was no test for this client